scripts/arangodb-docker-container-setup/arangodb-docker-container-setup.sh

```diff
@@ -183,6 +186,9 @@ wait
 
 echo "Starting new fresh instance of [$ARANGO_DB_SERVER]"
 
+# TODO support ssl for arango docker container
```

Comment:
Could we add a warning in the readme.md that this script is intended to be run locally as it doesn't use SSL?

Reply:
This will depend on what Sergio comes back with.  It sounds like the configuration for SSL is in the Arango docker image, but he said he'd get it all set for us to use with SSL.